0009-simple-payment-setup-protocol/0009-simple-payment-setup-protocol.md

```diff
@@ -146,40 +113,24 @@ The SPSP Client understands the following Cache-Control directives:
 
 ##### Response Body
 
-The response body is a JSON object. The fields provided are different depending on the `type` value of the object. Possible values for `type` are:
-
-`payee`
-: This is a general receiving account for peer-to-peer payments.
-
-`invoice`
-: This is an invoice, meaning it can be paid only once and only with a specific amount.
-
-##### Payee
-
-Payee information consists of basic account details. Amounts are chosen by the sender.
-
-Example Receiver:
-``` json
-{
-  "type": "payee",
-  "account": "ilpdemo.red.bob",
-  "currency_code": "USD",
-  "currency_symbol": "$",
-  "name": "Bob Dylan",
-  "image_url": "https://red.ilpdemo.org/api/receivers/bob/profile_pic.jpg"
-}
-```
+The response body is a JSON object that includes basic account details necessary for setting up payments.
 
 | Field | Type | Description |
 |---|---|---|
-| `type` | `"payee"` | Receiver type |
-| `account` | ILP Address | ILP Address of the recipient's account |
-| `currency_code` | String | Currency code to identify the receiver's currency. Currencies that have [ISO 4217](https://en.wikipedia.org/wiki/ISO_4217) codes should use those. Sender UIs SHOULD be able to render non-standard codes |
-| `currency_symbol` | String | Symbol for the receiver's currency intended for display in the sender's UI (e.g. `"$"` or `"shares"`). Sender UIs SHOULD be able to render non-standard symbols |
-| `name` | String | Full name of the individual, company or organization the receiver represents |
-| `image_url` | HTTPS URL | URL that a picture of the recipient can be fetched from. The image MUST be square and SHOULD be 128x128 pixels. |
-
-If this receiver is not available, an error can be generated at this stage:
+| `destination_account` | [ILP Address](../0015-ilp-addresses/0015-ilp-addresses.md) | ILP Address of the receiver's account |
+| `shared_secret` | 32 bytes, [base64-url encoded](https://en.wikipedia.org/wiki/Base64#URL_applications) | The shared secret to be used in the [Pre-Shared Key protocol](../0016-pre-shared-key/0016-pre-shared-key.md) |
+| `maximum_destination_amount` | Integer String | Maximum amount, denoted in the minimum divisible units of the ledger, the receiver will accept. This amount may be determined by the ledger's maximum account balance or transfer amount. If the receiver expects a specific amount to be delivered, this may be set equal to the `minimum_destination_amount` |
```

Comment:
Perhaps it's worth (re-)mentioning why we use integer strings instead of native JSON numbers:

> **Note:** Currency amounts are denominated as integer strings instead of native JSON numbers to avoid losing precision during JSON parsing. Applications MUST represent these numbers in a data type that has precision equal or greater than an unsigned 64-bit integer.